doc/website/blog/2023-04-20-terraform-v047.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "Manage database schemas with Terraform in plain SQL"
+authors: rotemtam
+tags: [schema, migration, terraform]
+---
+:::info TL;DR
+
+You can now use plain SQL to define the desired state of your database schema.
+  
+[See an example](#managing-database-schemas-in-plain-sql-using-terraform)
+
+:::
+
+Earlier this week, we released [v0.4.7](https://github.com/ariga/terraform-provider-atlas/releases/tag/v0.4.7) 
+of the [Atlas Terraform Provider](https://registry.terraform.io/providers/ariga/atlas/latest/docs).
+This release includes two changes that, in my opinion, are a  great improvement to the 
+experience of working with the Atlas Provider. 
+
+In this post, I will discuss these two changes and how they can help you to manage your database
+schemas with Terraform:
+
+* Support for the `docker://` driver for dev-databases.
+* Support for defining the desired state of your database schema in plain SQL (and
+  any other [schema loader](/blog/2023/03/13/atlas-v010#schema-loaders) supported by Atlas).
+
+### Improving the dev-database experience with the `docker://` driver
+
+Atlas relies on a connection to an empty database which it can use to perform
+various calculations and operations. This database is called the "[dev-database](/concepts/dev-database)",
+and it allows Atlas to do things like validate the correctness of user-provided code as well as
+normalize user-input to the way the database _actually_ sees it.
+
+In previous versions of the Atlas Provider, the dev-database needed to be provided
+by the user. This was a bit cumbersome, as the user needed to spin up a database (usually
+by running a local Docker container), and then provide the connection string to it
+in the `dev_url` field of the `atlas_schema` resource.
+
+To improve this experience, we added support for the `docker://` driver, which allows
+the user to only provide the database engine and version, and Atlas will spin up an
+ephemeral container with the correct database engine and version. In addition, starting
+this version, users may define the `dev_url` on the provider scope. For example:
+
+```hcl
+provider "atlas" {
+  dev_url = "docker://mysql/8/myapp"
+}
+```
+
+### Defining the desired state of the database schema in plain SQL
+
+In earlier versions of the Atlas Provider, the `atlas_schema` resource required the user
+to provide an [Atlas HCL](/atlas-schema/sql-resources) file which describes the desired
+state of the database schema. Many users found this syntax, which resembles Terraform's
+own, to be clean and concise. However, others disliked it and asked for a way to define
+the desired state in plain SQL.
```

Comment:
Dislike but as well limited support on HCL. With SQL, we now support everything SQL does support out if the box, which atlas does not yet.